public-docker-images/Dockerfile-debian-sid

```diff
@@ -1,54 +1,18 @@
-FROM golang:1.17 AS base
-LABEL maintainer="The Stripe Observability Team <support@stripe.com>"
-
-
-FROM base AS deps
-RUN mkdir -p /build
-ENV GOPATH=/go
-RUN apt-get update && apt-get install -y zip ca-certificates
-RUN go get -u -v github.com/ChimeraCoder/gojson/gojson
-RUN go get -u -v github.com/golang/protobuf/protoc-gen-go
-RUN go get -d -v github.com/gogo/protobuf/protoc-gen-gofast
-WORKDIR /go/src/github.com/gogo/protobuf
-RUN git fetch
-RUN git checkout v1.2.1
-RUN go install github.com/gogo/protobuf/protoc-gen-gofast
-WORKDIR /go
-RUN go get -u github.com/golang/dep/cmd/dep
-RUN go get -u -v golang.org/x/tools/cmd/stringer
-RUN wget https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip
-RUN unzip protoc-3.1.0-linux-x86_64.zip
-RUN cp bin/protoc /usr/bin/protoc
-RUN chmod 777 /usr/bin/protoc
-
-
-FROM deps AS src
-ARG BUILD_REF=master
-RUN mkdir -p /go/src/github.com/stripe
-WORKDIR /go/src/github.com/stripe
-RUN git clone --depth 1 https://github.com/stripe/veneur
-WORKDIR /go/src/github.com/stripe/veneur
-RUN git fetch --depth 1 origin ${BUILD_REF}:docker/${BUILD_REF}
-RUN git checkout docker/${BUILD_REF}
-
-
-FROM src AS test
-RUN go test -race -v -timeout 60s -ldflags "-X github.com/stripe/veneur.VERSION=$(git rev-parse HEAD) -X github.com/stripe/veneur.BUILD_DATE=$(date +%s)" ./...
```

Comment:
The new Docker images do not run tests. Arguably it was never necessary to run tests with the old version of the Dockerfile, given that we cloned from the latest head of master every time, but should we be running tests in the new version of the Dockerfile?